app/src/main/java/com/asfoundation/wallet/ui/iab/payments/common/error/IabErrorPresenter.kt

```diff
@@ -36,5 +47,27 @@ class IabErrorPresenter(
     )
   }
 
+  private fun handleSupportClick() {
+    disposables.add(Observable.merge(view.getSupportIconClicks(), view.getSupportLogoClicks())
+        .throttleFirst(50, TimeUnit.MILLISECONDS)
+        .flatMapCompletable { showSupport() }
+        .subscribe({}, { it.printStackTrace() })
+    )
+  }
+
+  fun showSupport(): Completable {
+    return walletService.getWalletAddress()
+        .flatMapCompletable { address ->
+          return@flatMapCompletable gamificationRepository.getUserStats(address)
```

Comment:
I disagree with this one. Maybe we should have a team discussion and reach an agreement to avoid going back and forth on the PRs